test/test_helper.exs

```diff
@@ -53,4 +56,10 @@ defmodule RollbarAPI do
   def call(conn, _test) do
     send_resp(conn, 404, "Not Found")
   end
+
+  defp with_delay(fun, delay \\ 100) do
```

Comment:
Do you think that
```
    r = Cowboy.shutdown(__MODULE__.HTTP)
    :timer.sleep(100)
    r
```
Looks better, than refactored to wrapped function which called twice?